ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```
@@ -300,23 +300,25 @@ cp *.csv *.xml *.json *.txt *.html "${ARTIFACT_DIR}/" 2>/dev/null || true
 
 # Experimental: run orion with original e-divisive binary (safe block, never breaks main execution)
 (
-    EXP_DIR="${ARTIFACT_DIR}/orion-original-edivisive"
+    EXP_DIR="/tmp/orion-original-edivisive"
     mkdir -p "$EXP_DIR"
-    EXP_BINARY="/tmp/orion-original-edivisive"
+    pushd "$EXP_DIR"
+    git clone -q --branch orig-edivisive-exp $ORION_REPO --depth 1
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)